src/document.rs

```diff
@@ -95,6 +97,8 @@ fn value_to_string(value: &Value) -> String {
         Value::JsonObject(json_object) => {
             serde_json::to_string(&json_object).unwrap()
         }
+        Value::Bool(b) => format!("{}", b),
+        Value::IpAddr(i) => format!("{}", *i),
```

Comment:
Nitpick, clippy now recommends using the new formatting method `format!("{b}")` for example.

Reply:
Linted with clippy